src/main/kotlin/rys/ajaxpetproject/repository/UserDAO.kt

```diff
@@ -0,0 +1,16 @@
+package rys.ajaxpetproject.repository
+
+import org.bson.types.ObjectId
+import rys.ajaxpetproject.model.MongoUser
+
+interface UserDAO {
+    fun save(mongoUser: MongoUser): MongoUser
+
+    fun getUserById(id: ObjectId): MongoUser?
```

Comment:
```suggestion
    fun findUserById(id: ObjectId): MongoUser?
```
get - means result or error, find - result or null, so if you return null when nothing is found - its find... method

Reply:
Done.